int16 distanceX
int16 distanceY
int16 soccerX
int16 soccerY
int16 soccerXMin
int16 soccerXMax
int16 soccerYMin
int16 soccerYMax
bool seesoccer
int16 soccersize
bool turnbody
